ci/scripts/driver.sh

```diff
@@ -17,65 +18,46 @@
 #################################################################
 # TODO using static build for GitHub CLI until fixed in HPC-Stack
 #################################################################
-GH=/home/Terry.McGuinness/bin/gh
-repo_url=${repo_url:-"https://github.com/NOAA-EMC/global-workflow.git"}
+export GH=${HOME}/bin/gh
 
 ################################################################
 # Setup the reletive paths to scripts and PS4 for better logging 
 ################################################################
-WF_ROOT_DIR="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
+HOMEGFS_DIR="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
 scriptname=$(basename "${BASH_SOURCE[0]}")
 echo "Begin ${scriptname} at $(date -u)" || true
 export PS4='+ $(basename ${BASH_SOURCE})[${LINENO}]'
 
-
-usage() {
-  set +x
-  echo
-  echo "Usage: $0 -h"
-  echo
-  echo    "  -h  display this message and quit"
-  echo
-  echo "This is top level script to run CI tests on the global-workflow repo"
-  if [[ -n "${TARGET+x}" ]]; then
-     echo "on the DEFAULT: ${TARGET} machine"
-  fi  
-  echo
-  exit 0
-}
-
-
 #########################################################################
 #  Set up runtime environment varibles for accounts on supproted machines
 #########################################################################
 
-source "${WF_ROOT_DIR}/ush/detect_machine.sh"
-if [[ "${MACHINE_ID}" != "UNKNOWN" ]]; then
-  TARGET="${MACHINE_ID}"
-else
-  echo "Unsupported platform. Exiting with error."
-  exit 1
-fi
-
-case ${TARGET} in
+source "${HOMEGFS_DIR}/ush/detect_machine.sh"
```

Comment:
```suggestion
source "${HOMEgfs}/ush/detect_machine.sh"
```
Might as well call `${HOMEgfs}/ush/module-setup.sh` here.